src/arrays/referencing_array_and_its_elements/hard/q11/MaxFinder.java

```diff
@@ -0,0 +1,34 @@
+package arrays.referencing_array_and_its_elements.hard.q11;
+
+/*
+Write a Java program that asks the user to enter 10 double values and stores them in an array. The array should then be
```

Comment:
The number of inputs in this question needs to be increased, and the question changed so that users actually need to store every variable instead of just changing out one whenever a higher on comes along.